src/lib/button/_button-theme.scss

```diff
@@ -107,7 +107,7 @@
 }
 
 @mixin mat-button-typography($config) {
-  .mat-button, .mat-raised-button, .mat-icon-button {
+  .mat-button, .mat-raised-button, .mat-icon-button, .mat-fab, .mat-mini-fab {
```

Comment:
I don't remember, but it was probably because the fabs aren't really meant to be used with text.